quesma/comment_metadata/comment_metadata.go

```diff
@@ -0,0 +1,76 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package comment_metadata
+
+import (
+	"fmt"
+	"net/url"
+	"regexp"
+)
+
+const ElasticFieldName = "fieldName"
+
+const commentMetadataVersion = "1"
```

Comment:
```suggestion
const metadataVersion = "1"
```
I think in this place `comment` is a little confusing